CHANGELOG.md

```diff
@@ -1,3 +1,71 @@
+### citus v5.2.0 (August 15, 2016) ###
+
+* Drops support for PostgreSQL 9.4; PostgreSQL 9.5 is required
+
+* Adds support for schema use in queries and functions
+
+* Evaluates non-immutable functions on master in INSERT and UPDATE commands
```

Comment:
> I think DELETE (WHERE clause) as well.

I'll re-read the code review, since that wasn't mentioned in any description (but INSERT and UPDATE were explicitly described).